Gemfile

```diff
@@ -11,3 +11,4 @@ end
 
 gem 'simplecov', require: false, group: :test
 gem 'mutant-rspec'
+gem 'xml-simple'
```

Comment:
Could you add this dependency to gemspec?
https://github.com/hanami/events/blob/master/hanami-events.gemspec#L28